t/Keybase.t

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Keybase)],
+    'keybase jbx' => test_spice(
+        '/js/spice/keybase/jbx',
```

Comment:
@mwolny Also be sure to update your tests and add some more for the different services.